src/utils/resources/vm/utils/selectors.ts

```diff
@@ -146,8 +147,11 @@ export const getWorkload = (vm: V1VirtualMachine): string =>
 export const getAccessCredentials = (vm: V1VirtualMachine): V1AccessCredential[] =>
   vm?.spec?.template?.spec?.accessCredentials;
 
-export const getIsDynamicSSHInjectionEnabled = (vm: V1VirtualMachine): boolean =>
-  getLabel(vm, DYNAMIC_CREDENTIALS_SUPPORT) === 'true' &&
+export const getIsDynamicSSHInjectionEnabled = (
+  vm: V1VirtualMachine,
+  bootableVolume?: BootableVolume,
```

Comment:
Sorry if I mislead you earlier,
What I initially meant is I want this function to look like this:
```
export const getIsDynamicSSHInjectionEnabled = (obj: V1VirtualMachine | BootableVolume): boolean =>
getLabel(obj, DYNAMIC_CREDENTIALS_SUPPORT) === 'true' &&
  Boolean(getAccessCredentials(vm)?.[0]?.sshPublicKey?.propagationMethod?.qemuGuestAgent);
 ```